app/src/main/java/com/money/manager/ex/home/MainActivity.java

```diff
@@ -1412,4 +1436,116 @@ private void showSelectDatabaseActivity() {
         intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
         startActivity(intent);
     }
+
+    private ArrayList<DrawerMenuItem> getGeneralReportGroupDrawerMenuItems() {
+
+        UIHelper uiHelper = new UIHelper(this);
+        int iconColor = uiHelper.getSecondaryTextColor();
+        ArrayList<DrawerMenuItem> childReportGroup = new ArrayList<>();
+
+        // Db setup
+        MmxOpenHelper MmxHelper = new MmxOpenHelper(this, new AppSettings(this).getDatabaseSettings().getDatabasePath());
```

Comment:
I think this is not the best way to get dB data.

Create a content provider and register with Uri, so you can use mmx infrastructure to get data